src/reactnative/builds/algoliasearch.reactnative.js

```diff
@@ -24,6 +24,12 @@ function algoliasearch(applicationID, apiKey, opts) {
 
   opts._ua = opts._ua || algoliasearch.ua;
 
+  opts.timeouts = opts.timeouts || {
+    connect: 2 * 1000,
+    read: 4 * 1000,
```

Comment:
Yes, front-end facing API clients are meant to be used in situations with a low hitsPerPage. Not like a backend client, thus we can lower down the read timeout